spec/controllers/api/v1/assignments_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+module Api
+    module V1
+
+        RSpec.describe AssignmentsController, type: :controller do
```

Comment:
```suggestion
        describe AssignmentsController, type: :controller do
```
Just to keep it consistent, though both do the same thing.